ui/imodel-components/src/ui-imodel-components/timeline/TimelineComponent.tsx

```diff
@@ -43,6 +43,13 @@ export interface TimelineMenuItemProps {
   /** duration for the entire timeline to play */
   timelineDuration: number;
 }
+/** TimelineDateMarkerProps: Mark a date on the timeline with an indicator
+ * @public
+ */
+export interface TimelineDateMarkerProps {

```

Comment:
Please change the structure of the comments on the props in TimelineDateMarkerProps and TimelineComponentProps so they end up in the API docs.

Reply:
Done!